config/default/editor.editor.basic_html.yml

```diff
@@ -55,13 +55,13 @@ settings:
             - Redo
             - Maximize
   plugins:
+    stylescombo:
+      styles: "a.decanter-button|Primary Button\r\na.decanter-button--big|Big Primary Button\r\na.decanter-button--secondary|Secondary Button\r\na.decanter-more-link|More Link\r\na.decanter-external-link|External Link\r\np.decanter-font-lead|Lead Font\r\np.decanter-font-splash|Splash Font\r\np.decanter-short-line-length|Short Line Length\r\np.decanter-caption|Caption\r\np.decanter-credits|Credits\r\nspan.underline-accent|Underlined Accent Text\r\ntable.decanter-table--borderless|Borderless Table"
```

Comment:
This doesn't apply the correct styling to match the mockups. I suggest changing the `span.underline-accent` to `a.underline-accent`. there should never be underlined text that isn't a link anyways (accessibility problems). see screenshot:
<img width="1349" alt="screen shot 2018-07-18 at 9 14 54 am" src="https://user-images.githubusercontent.com/7185045/42894210-23fe6f66-8a6b-11e8-8e19-9c8b38a44ecf.png">

Reply:
@pookmish I have made the change, but when I do so I don't see that style in any WYSIWYG dropdowns...